Making Find more browseable (#618)
DFE-Digital/bat-design-history: app/posts/find-teacher-training/2022-05-16-making-the-service-more-browseable.md

---
title: Making the service more browseable

> description: An exploration into how we might use links instead of form-based questions

Reviewer: ```suggestion
description: How we could use links instead of form-based questions.
```

date: 2022-05-16
ogImage:
  src: /find-teacher-training/making-the-service-more-browseable/todo.png
  alt: TODO
---

![](flow-update.png "")

We explored some alternative designs for the initial flow of the service, with the aim to make more of a browsing experience.

## The issue

We ran a diary study with participants who were early in the process of deciding whether training to become a teacher was right for them, and if so which courses to consider.

> When encountering the Find website, several participants made comments suggesting that the current design, where they are asked a series of questions before seeing matching courses, was off-putting or didn't work for them.

Reviewer: ```suggestion
When encountering the Find website, several participants made comments suggesting that the current design, where they are asked a series of questions before seeing matching courses, was off-putting or did not work for them.
```

Some of the reasons for this were:

* they did not have a specific location in mind, as they were willing to move
* the questions made them worry they were about to start an application, but were not yet ready to commit
* they didn't yet understand how the process of finding and applying for a course worked
* the GOV.UK branding looked 'official' (like submitting taxes) compared to the marketing websites

## A hypothesis

> We developed a hypothesis that switching from using form-based components (radios, checkboxes and button) to using plain content, links and stand-alone buttons would give users more confidence to navigate between pages without feeling like they were making firm decisions.

Reviewer: ```suggestion
Our hypothesis is that switching from using form-based components (radios, checkboxes and button) to using plain content, links and stand-alone buttons may give users more confidence to navigate between pages without feeling like they were making firm decisions.
```

## What we changed

> We switched the questions about age group (primary or secondary), subject and location to use links and buttons instead of radio buttons or checkboxes.

Reviewer: ```suggestion
We removed the questions about age group (primary or secondary), subject and location. Instead, we embedded links and buttons into blocks of content. 
```

### First page

> We changed the first page of the service, so that instead of showing three radio button options, we had a lot more content and links on the page.

Reviewer: ```suggestion
We changed the first page of the service, so that instead of showing 3 radio button options, we had a lot more content and links on the page.
```

The first section of the page described the differences between training to teach in primary and secondary schools, including the age ranges (for those who might not be familiar with the English school system) and the different minimum entry requirements.

A green start now style button was added for each age group.

Below this, we added some content to guide users who may not yet have decided which age group to teach. This promoted the teacher training adviser and get school experience services. We also added some links to guidance for teaching age groups not covered by service.

> Finally we included some guidance for those who are already a school teacher, who may be eligible for other more direct qualification routes.

Reviewer: ```suggestion
Finally, we included some guidance for people who‘re already school teachers - who may be eligible to qualify as a teacher a different way.
```

![](find-before-and-after.png "Existing first page compared with the prototype new one")

### Secondary subjects

> For the secondary age group, we then then took users to a page listing all the different subjects available, divided into two lists depending on whether a bursary or scholarship was on offer. We added numbers in brackets to indicate how many courses were available for each subject.

Reviewer: ```suggestion
For the secondary age group, we then then took users to a page listing all the different subjects available, divided into 2 lists depending on whether a bursary or scholarship was on offer. We added numbers in brackets to indicate how many courses were available for each subject.
```
